dockerfiles/Dockerfile-coda-daemon

```diff
@@ -0,0 +1,26 @@
+FROM ubuntu:18.04
+# Run with `docker build --build-arg coda_version=<version>`
+ARG coda_version
+RUN echo "Building image with of version $coda_version"
+
+# Dependencies
+RUN apt-get -y update && \
+  DEBIAN_FRONTEND=noninteractive apt-get -y upgrade && \
+  DEBIAN_FRONTEND=noninteractive apt-get -y install \
+    strace \
+    dumb-init \
+    libssl1.1 \
+    libprocps6 \
+    libgmp10 \
+    libgomp1 \
+    libffi6 \
+    apt-transport-https \
+    ca-certificates && \
+  rm -rf /var/lib/apt/lists/*
+
+# coda package
+RUN echo "deb [trusted=yes] http://packages.o1test.net stable main" > /etc/apt/sources.list.d/coda.list \
```

Comment:
Can you make repo an ARG too?

It's about to change to 'release', not 'stable'.